src/init.cpp

```diff
@@ -1415,6 +1415,8 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                 cache_sizes.block_tree_db,
                                 cache_sizes.coins_db,
                                 cache_sizes.coins,
+                                false,
```

Comment:
Please use named argument style here

Reply:
> Please use named argument style here

This suggestion makes sense. But one way or the other I would also like to incorporate or follow up with b7c7f64dc86f0fcdf07cb1e765f7cccc3a3c8897 (+129/-210) [#23280 (comment)](https://github.com/bitcoin/bitcoin/pull/23280#issuecomment-984149564) which gets rid of these arguments.

Reply:
Sure, getting rid of them is even better.